avr-hal-generic/src/wdt.rs

```diff
@@ -66,6 +87,14 @@ impl<H, WDT: WdtOps<H>> Wdt<H, WDT> {
         Self { p, _h: PhantomData }
     }
 
+    pub fn rearm(&mut self, m: &WDT::MCUSR) {
```

Comment:
Please add documentation: What is this function used for? What effects does it have?